stories/02-features/05-Style-with-themes.stories.mdx

```diff
@@ -0,0 +1,216 @@
+  display: flex;
+  width: 90%;
+`;
+```
+
+We recommend a folder structure like this:
+```
+  my-irving-project\
+    themes\
+      container\
+        yellowSection.js
+      index.js
+
+```
+
+Next step: import your custom theme to the themes' folder `index.js` file:
+
+```js
+import * as yellowSection from './container/yellowSection';
+
+export default {
+  'irving/container': {
+    yellowSection,
+  }
+}
+
+```
+That's it! A lovely feature of Irving is that as you noodle with your styling, the app hot reloads the styling without the need to refresh your browser. Very handy. However, templates and template parts are not hot-reloaded, so save and refresh after you have applied a theme to your component.
+
+### Theming your own styled component
+If you have a custom component that makes use of Styles Components in your Irving project you can still leverage Irving's theming functionality via the [`withThemes`](https://github.com/alleyinteractive/irving/blob/master/packages/styled/components/hoc/withThemes.js) higher-order component (HOC) within the `@irvingjs/styled` package. Unlike the `withThemes` HOC in Irving core, the `styled` version is somewhat simpler. A component using this HOC might look like the following:
+
+```js
+import React from 'react';
+import withThemes from '@irvingjs/styled/components/hoc/withThemes';
+import * as defaultStyles from './themes/default';
+import * as customThemeStyles from './themes/customTheme';
+
+const MyComponent = (props) => {
+  const { theme } = props;
+  const { Wrapper, Content } = theme;
+
+  return (
+    <Wrapper>
+      <Content>
+        This is my component content
+      </Content>
+    </Wrapper>
+  );
+};
+
+const themeMap = {
+  default: defaultStyles,
+  custom: customThemeStyles,
+};
+
+export default withThemes(themeMap)(Container);
+```
+
+As illustrated above, the `withThemes` HOC requires only a theme mapping object in which keys are a theme name and values are objects containing theme components. This HOC will provide a `theme` prop to the component to which the HOC is applied. This prop is an object on which both keys and values are the same (the name of each component function).
+
+As you may have noted already, the `withThemes` prop for styled components doesn't actually _require_ or integrate directly with styled components at all—it merely expects each value of the theme object to be a React component. This means that, for most intents and purposes, you will probably just supply an object of styled components in your theme. However, you can get creative if you want to (and at your own risk) by supplying any React component at all and, as long as the component is named the same as one of the theme values used in the themed component itself, it will be applied as-is. Be sure to render `children` as necessary, otherwise you may end up with chunks of your component that don't render (unless this is what you want!).
+
+### Theming components using Irving Core's CSS
```

Comment:
Not right now, but I think we need to break this out into its own story in a new section for just styling.

Reply:
Agreed, I think I'm going to make four sections: styling with CSS, theming with CSS, styling with styled components, theming with styled components.